boards/arm/mikroe_clicker_2/mikroe_clicker_2.dts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2020 Trifork
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/f4/stm32f407Xg.dtsi>
+
+/ {
+	model = "MikroE clicker 2 for STM32";
+	compatible = "mikroe,stm32-e407", "st,stm32f407";
+
+	chosen {
+		zephyr,console = &uart4;
+		zephyr,shell-uart = &uart4;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,ccm = &ccm0;
+	};
+
+
+	leds {
+		compatible = "gpio-leds";
+		ld1: led_1 {
+			gpios = <&gpioe 12 GPIO_ACTIVE_HIGH>;
+			label = "User LD1";
+		};
+		ld2: led_2 {
+			gpios = <&gpioe 15 GPIO_ACTIVE_HIGH>;
+			label = "User LD2";
+		};
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		user_button_0: key_0 {
+			label = "Key PE0";
+			gpios = <&gpioe 0 GPIO_ACTIVE_LOW>;
+		};
+		user_button_1: key_1 {
+			label = "Key K1";
+			gpios = <&gpioe 10 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	aliases {
+		led0 = &ld1;
+		led1 = &ld2;
+		sw0 = &user_button_0;
+		sw1 = &user_button_1;
+		mikrobus-1-i2c = &i2c3;
+		mikrobus-1-spi = &spi3;
+		mikrobus-1-uart = &usart2;
+		mikrobus-2-i2c = &i2c2;
+		mikrobus-2-spi = &spi2;
+		mikrobus-2-uart = &usart3;
```

Comment:
These should not be aliases, but node labels. See e.g. https://github.com/zephyrproject-rtos/zephyr/blob/70ff1d73c42f2e174c49b1e217712b52ad6ae78b/boards/arm/lpcxpresso55s16/lpcxpresso55s16_common.dtsi#L138 for an example.